QEfficient/compile/compile_helper.py

```diff
@@ -24,10 +24,15 @@ def create_and_dump_specializations(
                 "batch_size": str(batch_size),
                 "seq_len": str(prompt_len),
                 "ctx_len": str(ctx_len),
-            },
-            {"batch_size": str(batch_size), "seq_len": "1", "ctx_len": str(ctx_len)},
+            }
         ]
     }
+
+    # To handle repetative input in specializations when prompt_len is 1
+    if prompt_len != 1:
```

Comment:
in this case it will throw out of index error in line number 39 and 40
        specializations["specializations"][1]["full_batch_size"] = str(full_batch_size)
        specializations["specializations"][1]["batch_size"] = str(full_batch_size)
I believe the better approach would be to remove it at the end, otherwise every time when we make change for FBS or SpD we need to account for this change.

Reply:
Agree, I have updated the script by adding if condition while updating specializations for full batch size.